.github/ISSUE_TEMPLATE/2_bug_report_linux.yml

```diff
@@ -0,0 +1,108 @@
+name: "Report a Linux Issue"
+description: "Create a report for a runtime related Linux Issue"
+labels: ["os:linux"]
+body:
+- type: textarea
+  attributes:
+      label: "Important Information"
+      description: >
+          - mpv version
+           - Linux Distribution and Version
+           - Source of the mpv binary
+           - If known which version of mpv introduced the problem
+           - Window Manager and version
+           - GPU model, driver and version
+           - Possible screenshot or video of visual glitches
+
+          If you're not using git master or the latest release, update.
+          Releases are listed [here](https://github.com/mpv-player/mpv/releases).
+      value: |-
+          - mpv version:
+          - Linux version:
+          - Source:
+          - Introduced in version:
+          - Window Manager and version:
+          - GPU model, driver and version:
```

Comment:
This is scary for some to gather such information. Maybe we should provide commands to copy?

Reply:
We could copycat from mesa https://gitlab.freedesktop.org/mesa/mesa/-/blob/main/.gitlab/issue_templates/Bug%20Report.md#system-information

Actually, we already print this info in log, when things are initialized. So maybe it doesn't have to be so much strict to add all info. I will let you decide.

Reply:
some question, since it might not be everything we need for linux.

is this correct?
Window Manager and version > Xserver Version
GPU model, driver and version > GPU + Mesa Version
Linux Distribution and Version > Linux Version + Kernel Version

do we need something for gpu driver when using nvidia (proprietary) driver?
do we need a section for wayland and/or weston?

sry it's a bit hard for me to decide what is needed and important for linux issue reports.

Reply:
GPU model, driver and version > GPU + Mesa Version or some other proprietary nvidia drivers
Window Manager and version > you can leave it without suggestions, there is a lot WMs, based on waylad or X, whatever user provides will be fine
Linux Distribution and Version > Linux Version + Kernel Version

Reply:
are there any alternative commands? similar to the others we could add alternatives.